src/content/updates/migration/overview.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Migration
+tabs: ['Overview', 'Designers', 'Developers']
+---
+
+<AnchorLinks>
+
+- [Carbon X intro](#carbon-x-intro)
+- [About this guide](#about-this-guide)
+- [Rollout plan](#rollout-plan)
+- [Feedback](#feedback) 
+
+</AnchorLinks>
+
+## Carbon X
+
+Carbon X, aka v10, represents a complete refresh of the design system, based on the all-new [IBM Design Language](https://www.ibm.com/design/language/). While the look and feel are fresh, the basic functionality of Carbon's components has [deviated very little from v9](https://github.com/IBM/carbon-components/releases). Carbon X was engineered to provide a low-friction migration experience for users of Carbon v9.
+
+Carbon X is currently in Beta release. Please use this opportunity to explore and experiment with the new components, elements, layout patterns, and experimental UI shell. The Beta release is a work in progress; changes will be frequent and possibly breaking. **This release is not approved for production.** Please explore the system and [provide bountiful feedback](/help/support#github-issues)!
+
+## About this guide
+
+Note: this migration guide is for a future, unreleased version of Carbon. Anything in this document is subject to change up until v10 is released.
+
+This migration guide applies to teams who are currently using Carbon v9 in their products and migrating to Carbon X. For teams on older versions of Carbon, the migration process will be more technically involved. Some of the material in this guide will apply to those users, but not all of it is guaranteed to work in all migration scenarios. This guide contains specific migration guidance for both [designers](designers) and [developers](developers).
+
+Users not currently on Carbon v9 should refer to the [Getting started](/getting-started/about-carbon) section of this site for more general guidance on using Carbon X.
+
+## Rollout plan
+
+- **Mid-February 2019:** Beta release - current
+- **Mid-March 2019:** Production release
```

Comment:
For both of these, remove the "Mid-" because that isn't happening in production release.